pkg/controllers/managementuser/rbac/project_handler.go

```diff
@@ -87,6 +95,17 @@ func (p *pLifecycle) Remove(project *v3.Project) (runtime.Object, error) {
 				}
 			}
 		}
+
+		// remove project scoped secrets if they exist
+		secrets, err := p.secretClient.List(namespace.Name, metav1.ListOptions{LabelSelector: projectScopedSecretLabel + "=" + project.Name})
+		if err != nil {
+			return project, fmt.Errorf("failed to list project scoped secrets: %w", err)
+		}
+		for _, secret := range secrets.Items {
+			if err := p.secretClient.Delete(namespace.Name, secret.Name, &metav1.DeleteOptions{}); err != nil {
+				return project, fmt.Errorf("failed to delete project scoped secret %s: %w", secret.Name, err)
```

Comment:
Should we try to delete as many PSP's as possible instead of bailing out on the first error?